src/core/graph.js

```diff
@@ -535,12 +703,29 @@ export class Graph {
     return result;
   }
 
+  /**
+   * Produce a copy of a Graph.
```

Comment:
```
   * Produce a copy of a Graph.
```

…of _this_ graph, not just of _a_ graph.

```
   *
   * The copy is equal to the original, but distinct, so that they may be
   * modified independently.
```

Good.